layer/trace_layer.cpp

```diff
@@ -53,6 +53,58 @@ const VkLayerProperties kLayerProps = {
 
 const std::vector<VkExtensionProperties> kDeviceExtensionProps = { VkExtensionProperties{ "VK_EXT_tooling_info", 1 } };
 
+/// An alphabetical list of device extensions which we do not report upstream if
+/// other layers or ICDs expose them to us.
+const char* const kDeviceExtensionBlacklist[] = {
+    // Look problematic:
+    "VK_EXT_external_memory_dma_buf",
+    "VK_EXT_external_memory_host",
+    // No app can expect this to always be present and we wouldn't want them to make
+    // decisions based on low level PCI bus info about the capture device.
+    "VK_EXT_pci_bus_info",
+    // Looks problematic:
+    "VK_EXT_physical_device_drm",
+    // Look problematic:
+    "VK_KHR_external_memory_fd",
+    "VK_KHR_external_memory",
+    // Supporting the CPU moving around descriptor set data directly has too many
+    // perf/robustness tradeoffs to be worth it.
+    "VK_VALVE_descriptor_set_host_mapping",
```

Comment:
Can this be `VK_VALVE_DESCRIPTOR_SET_HOST_MAPPING_EXTENSION_NAME` ?

Reply:
done.